packages/types/package.json

```diff
@@ -11,6 +11,9 @@
   "engines": {
     "node": ">=12.18.0"
   },
+  "publishConfig": {
```

Comment:
So is `publishConfig` actually needed?

Reply:
Yeah, NPM gives an error about not being able to publish "private" packages without it. Found out when publishing `@cumulus/types`